lib/Doctrine/ORM/Event/PostRemoveEventArgs.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Event;
+
+use Doctrine\ORM\EntityManagerInterface;
+
+class PostRemoveEventArgs extends LifecycleEventArgs
+{
+    /** @param object $entity */
+    public function __construct($entity, EntityManagerInterface $objectManager, private mixed $identifier)
```

Comment:
Our codebase still maintains compatibility with PHP 7 and because of that we cannot use the `mixed` type yet.